### DISCUSSION TITLE
sdlf-team repository

### DISCUSSION THREAD

**Discussion by mninawa** [aws-solutions-library-samples/data-lakes-on-aws#51] [Q&A]
Hi Guys
i have this error when i ran the sdlf-team repo
<img width="1370" alt="Screenshot 2021-07-14 at 05 12 00" src="https://user-images.githubusercontent.com/4068536/125555203-646506eb-295a-43d7-b8a3-73e63bd50970.png">

`Embedded stack arn:aws:cloudformation:eu-central-1:079486612340:stack/sdlf-engineering-rIAMStack-1HP776CJ2WGEE/70ce3191-e450-11eb-b944-06640ee0538e was not successfully created: The following resource(s) failed to create: [rTeamIAMManagedPolicy].`

what is it that i need to add on my side to get this to continue

**Comment by jaidisido** [comment]
Hi,

Pushing to the `sdlf-team` repository should have first triggered a CodeBuild project named `sdlf-cicd-team-repos-codebuild` which in turn should have created a stack called `sdlf-crossaccount-role-dev-engineering`:
![image](https://user-images.githubusercontent.com/16362107/125602634-6422d473-db7a-45ea-9572-dd659da427c9.png)

That stack seems to be missing in the picture you shared. In CodeBuild, could you please check if there was an error with that project execution (`sdlf-cicd-team-repos-codebuild`)?

**Reply by mninawa** [reply]
<img width="1311" alt="Screenshot 2021-07-14 at 17 32 25" src="https://user-images.githubusercontent.com/4068536/125649796-ad2890cc-4ebd-47ab-91d1-41f869cfa6f9.png">
i have it created!

**Reply by mninawa** [reply]
i got it right this time... it working now... will update you once all is working

**Reply by mninawa** [reply]
im happy.. all is working..

**Reply by jaidisido** [reply]
Nice! Glad to hear it all worked 🙂
